tests/test_pipeline_utils.py

```
@@ -404,3 +406,54 @@ def test_non_sandbox_command_mounts_unchanged():
+    cluster_config = {
+        "executor": "local",
+        "containers": {"sandbox": "sandbox:latest"},
+    }
+
+    add_task(
+        exp=exp,
+        cmd="echo hello",
+        task_name="test-task",
+        cluster_config=cluster_config,
+        container="main:latest",
+        log_dir="/tmp/logs",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/logs"

(S108)